packages/x-date-pickers/src/internals/components/FakeTextField/FakeTextField.tsx

```diff
@@ -1,23 +1,32 @@
 import * as React from 'react';
 import Stack from '@mui/material/Stack';
-import { FieldSection } from '../../../models';
+
+export interface FakeTextFieldElement extends React.HTMLAttributes<HTMLDivElement> {
+  before: string;
+  after: string;
+}
+
+export interface A {}
```

Comment:
I might be missing something, but do we need this? 
```suggestion

```

Reply:
Oops that's a test I forgot to remove :laughing: